_data/master/navbars/introduction.yml

```diff
@@ -5,3 +5,7 @@ order: 0
 toc:
 - title: Introduction
   path: /introduction/
+- title: Component Architecture
```

Comment:
How about we move them under `/introduction/containers/*` directory but still leave them at the toplevel of the sidebar until we introduce the Openstack introduction?